zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -42,23 +42,20 @@ class ElasticsearchClient implements AutoCloseable {
   private final TemplateReader templateReader;
   private final RecordIndexRouter indexRouter;
   private final BulkIndexRequest bulkIndexRequest;
+  private final MeterRegistry meterRegistry;
 
   private ElasticsearchMetrics metrics;
 
-  ElasticsearchClient(final ElasticsearchExporterConfiguration configuration) {
```

Comment:
:question: Was this unused?

Reply:
It was used however since it was for an actual instantiation a meter registry is needed to be passed in so it is not fit for purpose